packages/wrangler/src/proxy.ts

```diff
@@ -421,6 +427,9 @@ export async function waitForPortToBeAvailable(
     }
 
     function checkPort() {
+      // Will clear the interval and timeout if signal is called,
+      // using resolve prevents Error messages on the console.
+      options.abortCtrl?.aborted && doResolve();
```

Comment:
let's wrap this in a conditional 
```suggestion
      if(options.abortCtrl?.aborted){
        doResolve();
      }
```